website/docs/d/private_dns_resolver_forwarding_rule.html.markdown

```diff
@@ -0,0 +1,56 @@
+---
+subcategory: "Private DNS Resolver"
+layout: "azurerm"
+page_title: "Azure Resource Manager: azurerm_private_dns_resolver_forwarding_rule"
+description: |-
+  Gets information about an existing Private DNS Resolver Forwarding Rule.
+---
+
+# Data Source: azurerm_private_dns_resolver_forwarding_rule
+
+Gets information about an existing Private DNS Resolver Forwarding Rule.
+
+## Example Usage
+
+```hcl
+data "azurerm_private_dns_resolver_forwarding_rule" "example" {
+  name                      = "example-rule"
+  dns_forwarding_ruleset_id = "example-forwarding-rulset-id"
+}
+```
+
+## Arguments Reference
+
+The following arguments are required:
```

Comment:
To maintain consistency with other docs, this should be 
```suggestion
The following arguments are supported:
```

Reply:
done